sites/uat/features/high-value/stanford_sites_helper.feature

```diff
@@ -5,33 +5,27 @@ Feature: Stanford Sites Helper Module
   I want to ensure the Stanford Sites Helper module is working properly
 
   @api @safe @deploy @javascript
-  Scenario Outline: Homepage text content
```

Comment:
For each line, it goes through and does each step in the outline:
1. Given I am an anonymous user
2. And I am on the homepage
3. Then I should see " Once you have logged in, you will be able to remove this text from your homepage" in the "Content Body" region
1. Given I am an anonymous user
2. And I am on the homepage
3. Then I should see "Create a new page on your site" in the "Content Body" region

etc.

Not so bad for this one, but for the further ones, where it's creating a new user, logging in as that user, waiting for AJAX to finish, it's doing that for every line in the "Examples" table.